samples/nrf9160/cloud_simple/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+CONFIG_LOG=y
+CONFIG_LOG_IMMEDIATE=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_NBIOT=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+CONFIG_LTE_LINK_CONTROL_LOG_LEVEL_DBG=y
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+
+# Generic cloud API
+CONFIG_CLOUD_API=y
+
+# AWS IoT library
+CONFIG_AWS_IOT=y
+CONFIG_AWS_IOT_LOG_LEVEL_DBG=y
+
+# nRF Cloud
+CONFIG_NRF_CLOUD=y
+CONFIG_NRF_CLOUD_LOG_LEVEL_DBG=y
+
+# Heap and stacks
+CONFIG_HEAP_MEM_POOL_SIZE=16384
```

Comment:
Do we need that much in this simple sample, could we try to profile it?

Reply:
The heap size was way to exaggerated, used during development. Changed the heap memory size 8192.